packages/admin-ui/client/pages/List/Search.js

```diff
@@ -0,0 +1,83 @@
+/** @jsx jsx */
+
+import { jsx } from '@emotion/core';
+import { useRef, useState } from 'react';
+
+import { SearchIcon, XIcon } from '@arch-ui/icons';
+import { Input } from '@arch-ui/input';
+import { A11yText } from '@arch-ui/typography';
+import { LoadingSpinner } from '@arch-ui/loading';
+import { colors } from '@arch-ui/theme';
+
+import { useListSearch } from './dataHooks';
+
+export default function Search({ isLoading, list }) {
+  const { searchValue, onChange, onClear, onSubmit } = useListSearch(list.key);
+  const [value, setValue] = useState(searchValue);
+  const ref = useRef();
```

Comment:
I'd call this `inputRef` so it's clear what element this ref is for.

Reply:
Sure, no worries